src/vs/workbench/contrib/webview/browser/pre/main.js

```diff
@@ -443,7 +448,7 @@
 					});
 
 					// Bubble out link clicks
-					newFrame.contentWindow.addEventListener('click', handleInnerClick);
+					newFrame.contentWindow.addEventListener('mousedown', handleInnerClick);
```

Comment:
Why is this needed?

Reply:
Because the click event only fires on left mouse button clicks. To prevent the middle mouse button from opening broken links in the browser, I needed to have an event that fires on middle clicks as well.